src/frequenz/client/base/channel.py

```diff
@@ -0,0 +1,75 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Handling of gRPC channels."""
+
+from urllib.parse import parse_qs, urlparse
+
+from grpclib.client import Channel
+
+
+def _to_bool(value: str) -> bool:
```

Comment:
no docs?

Reply:
It's private and trivial, I intentionally left it without docs, but I can add some if you think it should have some.